editor/project_manager.cpp

```diff
@@ -2535,7 +2535,7 @@ ProjectManager::ProjectManager() {
 	}
 
 	// TRANSLATORS: This refers to the application where users manage their Godot projects.
-	DisplayServer::get_singleton()->window_set_title(VERSION_NAME + String(" - ") + TTR("Project Manager"));
+	DisplayServer::get_singleton()->window_set_title(VERSION_NAME + String(" - ") + TTR("Project Manager", "Application"));
```

Comment:
That one doesn't seem to work (with the `3.x` version):
```diff
@@ -5002,8 +5399,7 @@ msgstr ""
 msgid "Editor SSL Certificates"
 msgstr ""
 
-#. TRANSLATORS: This refers to the application where users manage their Godot projects.
-#: editor/editor_settings.cpp editor/project_manager.cpp
+#: editor/editor_settings.cpp
 msgid "Project Manager"
 msgstr ""
 
```

Reply:
Or more like, there's another one in `editor/editor_settings.cpp` to change.

Reply:
That's part of a property path... I was about to add context support for property path :)

But I think it'll be delayed after the inspector toggle. I'll remove the context for this one.